pkg/beacon/relay/integration_test.go

```diff
@@ -165,6 +168,66 @@ func TestInactiveMemberPublicKeySharesReconstructionAndSigning(t *testing.T) {
 	}
 }
 
+func TestInvalidSignatureSharesExists(t *testing.T) {
```

Comment:
Maybe `TestsSigningWithInvalidSignatureShare`? I think this one better fits the rest of the names we have in integration tests.